go.mod

```diff
@@ -10,8 +10,8 @@ require (
 	github.com/aliyun/aliyun-oss-go-sdk v2.0.1+incompatible
 	github.com/coreos/go-systemd/v22 v22.1.0
 	github.com/frankban/quicktest v1.9.0 // indirect
-	github.com/gardener/etcd-druid v0.3.0
-	github.com/gardener/gardener v1.22.3
+	github.com/gardener/etcd-druid v0.5.0
+	github.com/gardener/gardener v1.23.0
```

Comment:
As part of vendoring `gardener/gardener@v1.23.0`, you can revert the changes from https://github.com/gardener/gardener-extension-provider-alicloud/pull/289 as `gardener/gardener@v1.23.0` contains the corresponding fix.